CHANGELOG.md

```diff
@@ -1,3 +1,10 @@
+# v0.7.0
+## Fail job when badly formatted CHANGELOG
+Before a trailing separator could fail the job for un unexpected reason.
```

Comment:
Typo `un`.

Perhaps:
`Previously, a trailing separator could fail the job for an unexpected reason.`